app/lib/features/cross_signing/views/request_ready.dart

```diff
@@ -49,23 +48,25 @@ class RequestReadyView extends StatelessWidget {
           const Spacer(),
           // TextButton(
           //   onPressed: () {},
-          //   child: const Row(
+          //   child: Row(
           //     mainAxisAlignment: MainAxisAlignment.center,
           //     children: [
-          //       Padding(
+          //       const Padding(
           //         padding: EdgeInsets.all(8),
           //         child: Icon(Atlas.camera),
           //       ),
-          //       Text('Scan with this device'),
+          //       Text(L10n.of(context).verificationScanWithThisDevice),
           //     ],
           //   ),
           // ),
           // const Spacer(),
```

Comment:
Let's remove commented code if not in use.

Reply:
Besides emoji, QR code scanning is another way to perform cross signing.
I left QR code for future implementation.